packages/docusaurus-plugin-google-gtag/src/types.d.ts

```diff
@@ -10,11 +10,7 @@
 interface Window {
   gtag: (
     command: string,
-    fields: string,
-    params: {
-      page_title?: string;
-      page_location?: string;
-      page_path?: string;
-    },
+    action: string,
+    params?: string | Gtag.EventParams,
```

Comment:
```suggestion
    fields: string,
    params: {
      page_title?: string;
      page_location?: string;
      page_path?: string;
    },
```

Hey, I'm going to revert this change because this PR does not install anymore the TS types. `Gtag.EventParams` is then not available. CI doesn't catch this because we don't actually have any website code using this API directly.

I would have edited your PR normally but it's opened from the branch "main" of your repo: it's not a good idea to do so and makes it harder for me, gh cli is not happy because I already have a "main" branch locally.

<img width="694" alt="CleanShot 2022-10-21 at 10 39 59@2x" src="https://user-images.githubusercontent.com/749374/197152610-a73bb363-78c8-4b16-9e24-72c753d5ec79.png">

Reply:
(found a way to checkout with the --branch option 👍  will edit and fix the pr)